cypress/plugins/index.js

```diff
@@ -298,7 +298,10 @@ module.exports = async (on, config) => {
             jsonReport = readDataFromFile(filePath + fileName);
           }
           const reportProperties = {};
+          const customFooter = require("../fixtures/customFooter.json");
```

Comment:
Can use cy.fixture.then() instead of require

Reply:
no, can't use cy in after:run because of ReferenceError: cy is not defined